packages/scandipwa/src/store/Wishlist/Wishlist.dispatcher.ts

```diff
@@ -49,8 +49,7 @@ export const isWishlistEnabled = (): boolean => {
 /**
  * Product Wishlist Dispatcher
  * @class WishlistDispatcher
- * @namespace Store/Wishlist/Dispatcher
- */
+ /* @namespace Store/Wishlist/Dispatcher */
```

Comment:
No need to make it a separate comment.
Or add a closing of the previous comment.

Reply:
+